addons/website_event/tests/test_event_internals.py

```diff
@@ -224,3 +226,87 @@ def test_registration_answer_search(self):
         # should fetch "registration_2" because the answer to the first question is "Q1-Answer2"
         # should fetch "registration_3" because the answer to the third question is "Answer2" (as free text)
         self.assertEqual(search_res, registration_2 | registration_3)
+
+    @users('portal')
+    def test_portal_user_is_visible_on_website(self):
```

Comment:
If methods are currently ordered alphabetically, we should respect that order.
We could rename to "test_website_visibility_portal" and "test_website_visibility_public" to keep them together + shorter.